core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairSchedulerImpl.java

```diff
@@ -127,7 +129,8 @@ private void createTableSchedule(TableReference tableReference, RepairConfigurat
 
         if (oldTableRepairJob != null)
         {
-            deleteTableSchedule(tableReference);
+            ScheduledJob job = myScheduledJobs.remove(tableReference);
```

Comment:
I think it would be better if the code is changed to:

```
TableRepairJob oldTableRepairJob = myScheduledJobs.remove(tableReference);

if (oldTableRepairJob != null)
{
  descheduleTableJob(oldTableRepairJob);
}
```

Reply:
Good catch!

Reply:
Just realized, we can remove the null check too